### PR TITLE
Fix multiple simultaneous open's for the browsers WebSocket implementation

### PR DESCRIPTION
**Pull request by vladyslavry141**
- [x] tests and linter show no problems (`npm t`)
- [x] code is properly formatted (`npm run fmt`)
- [x] description of changes is added in CHANGELOG.md

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,8 @@
 
 - Fix clients Map memory leak
 - Add static create method for server-side Client
+- Add open event
```
Comment (nechaido) [inline CHANGELOG.md:7]:
```suggestion
- Add `open` event to the browser implementation
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,8 @@
...
+- Fix multiple opening webqsockets
```
Comment (nechaido) [inline CHANGELOG.md:8]:
```suggestion
- Fix multiple simultaneous `open`'s for the browsers WebSocket implementation
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,8 @@
 
 - Fix clients Map memory leak
 - Add static create method for server-side Client
+- Add open and close event
```
Comment (nechaido) [inline CHANGELOG.md:7]: No `close` event was added here.

Comment (nechaido) [inline CHANGELOG.md:7]:
```suggestion
- Add `open` and `close` events to the browser implementation
```

**Review by nechaido — APPROVED** [review]

**Review by tshemsedinov — APPROVED** [review]

**Comment by tshemsedinov** [comment]
Landed in https://github.com/metarhia/metacom/commit/fd249b94fd87496952f0fb1ca7064d7d064b8e5c and https://github.com/metarhia/metacom/commit/a78c707df729fd479f6708ac7d76bf6ba6bed00b

**Pull request closed without merging**